spec/ransack/search_spec.rb

```diff
@@ -614,19 +614,19 @@ def remove_quotes_and_backticks(str)
         expect(@s.result.first.id).to eq 1
       end
 
-      it "PG's sort option", if: ::ActiveRecord::Base.connection.adapter_name == "PostgreSQL" do
+      it "fields sort option", if: ::ActiveRecord::Base.connection.adapter_name != "Mysql2" do
```

Comment:
If Arel does implement working `nulls_first()` and `nulls_last()` for MySQL, or if Ransack is ever updated to implement it, then we will need specific MySQL tests as the generated SQL will be different.